src/infra/AutocompleteServiceImpl.ts

```diff
@@ -70,8 +71,17 @@ export class AutocompleteServiceImpl implements AutocompleteService {
     };
 
     const response = 'token' in this.config
-      ? await this.httpService.get<ApiResponse>(this.universalEndpoint, queryParams, this.config.token)
-      : await this.httpService.get<ApiResponse>(this.universalEndpoint, queryParams);
+      ? await this.httpService.get<ApiResponse>(
```

Comment:
Is it possible to remove this ternary op? I know I originally removed the ternary op but not sure if I missed something wrong with doing so?

Reply:
I added the ternary operator back in because otherwise we have to check for `undefined` (or `expect.anything()`) as the `authToken` param for each unit test that doesn't specify a token

Reply:
ah I see, either is fine then! I was mostly wondering if the behavior didn't work as expected